pyscf/eph/rhf.py

```diff
@@ -144,22 +170,36 @@ def get_eph(ephobj, mo1, omega, vec, mo_rep):
         vcore.append(vtot)
     vcore = np.asarray(vcore).reshape(-1,nao,nao)
     mass = mol.atom_mass_list() * 1836.15
-    nmodes, natoms = len(omega), len(mass)
-    vec = vec.reshape(natoms, 3, nmodes)
-    for i in range(natoms):
-        for j in range(nmodes):
-            vec[i,:,j] /= np.sqrt(2*mass[i]*omega[j])
-    vec = vec.reshape(3*natoms,nmodes)
+    vec = _freq_mass_weighted_vec(vec, omega, mass)
     mat = np.einsum('xJ,xuv->Juv', vec, vcore)
     if mo_rep:
         mat = np.einsum('Juv,up,vq->Jpq', mat, mf.mo_coeff.conj(), mf.mo_coeff, optimize=True)
     return mat
 
 
 class EPH(rhf.Hessian):
-    def __init__(self, scf_method):
+    '''EPH for restricted Hartree Fock
+
+    Attributes:
+        cutoff_frequency : float or int
+            cutoff frequency in cm-1. Default is 80
+        keep_imag_frequency : bool
+            Whether to keep imaginary frequencies in the output.  Default is False
+
+    Saved results
+
+        omega : numpy.ndarray
+            Vibrational frequencies in au.
+        vec : numpy.ndarray
+            Polarization vectors of the vibration modes
+        eph : numpy.ndarray
+            Electron phonon matrix eph[j,a,b] (j in nmodes, a,b in norbs)
+    '''
+
+    def __init__(self, scf_method, **kwargs):
         rhf.Hessian.__init__(self, scf_method)
-        self.CUTOFF_FREQUENCY=CUTOFF_FREQUENCY
+        self.cutoff_frequency = kwargs.pop("CUTOFF_FREQUENCY", CUTOFF_FREQUENCY)
+        self.keep_imag_frequency = kwargs.pop("KEEP_IMAG_FREQUENCY", KEEP_IMAG_FREQUENCY)
```

Comment:
Better to put cutoff_frequency and keep_imag_frequency in __init__ argument list. The name of the arguments should be all lower cases